src/Main.java

```diff
@@ -1,9 +1,46 @@
+import java.util.Scanner;
 
 public class Main {
+
+    /**
+     * Main method to start the program.
+     *
+     * @param args The command line arguments.
+     */
     public static void main(String[] args) {
+        // numeriek array
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+
+        // alfabetisch array
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
+
+        // Maak een nieuwe instantie van de Translator class
+        Translator translator = new Translator(alphabetic, numeric);
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+        Scanner scanner = new Scanner(System.in);
+        boolean play = true;
+        String ongeldig = "ongeldige invoer";
 
+        // while loop om het spel te starten
+        while (play) {
+            System.out.println("Type 'x' om te stoppen \nType 'v' om te vertalen");
+            String input = scanner.nextLine();
 
+            if(input.equals("x")) {
+                play = false;
+            } else if (input.equals("v")) {
+                System.out.println("Type een cijfer in van 0 t/m 9");
+                int number = scanner.nextInt();
+                scanner.nextLine();
+                if (number < 10) {
```

Comment:
Je had hier eventueel nog kunnen toevoegen dat number ook groter of gelijk aan 0 moet zijn.

```suggestion
                if (numer >= 0 && number < 10) {
```

Reply:
You're right I can do that and check for negative numbers as well but since, I'm using getOrDefault in the Translator class it will take care of this case for me and return the default value which is `ongeldige invoer`. But I do agree with you it would be cleaner and better to cover that case here as well. I will apply the this changes.